lib/commands/implementations/unban.js

```diff
@@ -9,8 +9,8 @@ function unban(input, services) {
   if (_.isEmpty(userToUnban)) {
     return new CommandOutput(new Error('User was empty.'), `Something did not work. ${userToUnban} not unbanned.`);
   }
-  services.punishmentStream.write(makeUnban(userToUnban, `Unbanning ${userToUnban}`));
-  return new CommandOutput(null, null, null);
+  services.punishmentStream.write(makeUnban(userToUnban));
```

Comment:
This is left over from some refactoring I did, I think the change here should actually be to just get rid of the output from both makeUnmute and makeUnban, and have no output in unmute or unban. Neither twitch nor DGG supports reasons for unban/unmute, and they also both automatically output to the chat, so ouputting twice looks bad.